nginx-stable/snapcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: nginx
+version: "1.11.1"
+summary: nginx is an HTTP and reverse proxy server
+description: |
+  nginx ("engine x") is an HTTP and reverse proxy server, a mail proxy server,
+  and a generic TCP/UDP proxy server, originally written by Igor Sysoev.  For a
+  long time, it has been running on many heavily loaded Russian sites including
+  Yandex, Mail.Ru, VK, and Rambler.
+confinement: devmode
+
+apps:
+  nginx:
+    command: usr/nginx/nginx
+    stop-command: nginx -s quit
+    daemon: simple
```

Comment:
I'd remove the daemon line and I'd start the binary manually for debugging purposes until everything is working.